resourcestore/generate_typescript.go

```diff
@@ -52,17 +68,15 @@ func (s *Store) GenerateTypeScript(dst string) error {
 	}
 	defer f.Close()
 
-	tsFile, err := template.New("").Parse(tmpl)
-	if err != nil {
-		panic(err)
-	}
+	tsFile := template.Must(template.New("").Parse(tmpl))
 
 	if err := tsFile.Execute(f, TSGenerator{
-		Permissions: s.permissions(),
-		Resources:   s.resources(),
-		Mappings:    s.permissionResources(),
+		Permissions:         s.permissions(),
+		Resources:           s.resources(),
+		ResourceTags:        s.tags(),
+		ResourcePermissions: s.resourcePermissions(),
 	}); err != nil {
-		panic(err)
+		return err
 	}
```

Comment:
Our pattern is to wrap the errors that are coming out of external packages, with the function name. 

This is the typical pattern:
```go
	if err := tsFile.Execute(f, TSGenerator{
		Permissions:         s.permissions(),
		Resources:           s.resources(),
		ResourceTags:        s.tags(),
		ResourcePermissions: s.resourcePermissions(),
	}); err != nil {
		return errors.Wrap(err, "template.Template.Execute()")
	}
```